src/status_im/node/core.cljs

```diff
@@ -60,6 +60,15 @@
     (assoc :ListenAddr ":30304"
            :DataDir (str (:DataDir config) "_dev"))))
 
+(def login-node-config
+  {:WalletConfig (cond-> {:Enabled true}
+                   (not= config/opensea-api-key "")
+                   (assoc :OpenseaAPIKey config/opensea-api-key))})
+
+(defn- get-login-node-config [config]
+  (merge config
+         login-node-config))
```

Comment:
It would be fine to have it in a single line. Not a big deal though.

Reply:
Done.